Bottom: ec6a93d18955cc9b70e25eb5a517ceaa526ead0e
Top:    7d8ef5a633ac949323a57389063a1ff6f07106e7
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Exec account script on exim4 container

Instead of starting a new container to add accounts,
execute script within existing container.


---
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index fa74e754..3db11193 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -128,5 +128,5 @@
 # Email accounts
 
 - name: populate email passwords
-  shell: 'docker run --rm --volumes-from {{ exim4_dv_name }} {{ exim4_image_name }}:{{ docker_image_tag }} add_account {{ item.value.user }} "{{ item.value.password }}"'
+  shell: 'docker exec {{ exim4_container_name }} /docker-entrypoint.sh add_account {{ item.value.user }} "{{ item.value.password }}"'
   with_dict: '{{ email_accounts }}'
